tests/components/plex/test_config_flow.py

```diff
@@ -502,3 +530,161 @@ async def test_multiple_servers_with_import(hass):
         )
         assert result["type"] == "abort"
         assert result["reason"] == "non-interactive"
+
+
+async def test_manual_config(hass):
+    """Test creating via manual configuration."""
+
+    class WrongCertValidaitionException(requests.exceptions.SSLError):
+        """Mock the exception showing an unmatched error."""
+
+        def __init__(self):
+            self.__context__ = ssl.SSLCertVerificationError(
+                "some random message that doesn't match"
+            )
+
+    # Basic mode
+    result = await hass.config_entries.flow.async_init(
+        config_flow.DOMAIN, context={"source": "user"}
+    )
+
+    assert result["data_schema"] is None
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5462445)